apitools/base/py/batch.py

```diff
@@ -180,11 +181,14 @@ def Add(self, service, method, request, global_params=None):
             http_request, self.retryable_codes, service, method_config)
         self.api_requests.append(api_request)
 
-    def Execute(self, http, sleep_between_polls=5, max_retries=5):
+    def Execute(self, http, max_batch_size=None,
```

Comment:
Small thing: I'd say add the new argument at the end of the list, if you want to be friendly to potential callers passing all args positionally.